realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -281,6 +283,35 @@ public void refresh() {
     }
 
     /**
+     * Blocks the current thread util transactions are committed by other threads to this Realm or
+     * {@link #stopWaitForChange()} gets called.
```

Comment:
"{@link #stopWaitForChange()} gets called." -> "{@link #stopWaitForChange()} gets called from other threads." I think it is worth saying that `stopWaitForChange` is supposed to be called from another thread.